.github/workflows/ci-backend.yml

```diff
@@ -111,28 +111,65 @@ jobs:
         needs: changes
         if: ${{ needs.changes.outputs.backend == 'true' && github.repository == 'PostHog/posthog' }}
 
-        name: Django tests – Py ${{ matrix.python-version }}
+        name: Django tests – Py ${{ matrix.python-version }} ${{ matrix.name }} (${{matrix.group}}/${{ matrix.total-concurrency }})
         runs-on: ubuntu-latest
 
         strategy:
             fail-fast: false
             matrix:
-                python-version: ['3.8.5', '3.9.0']
+                python-version: ['3.8.5']
+                ee: [true]
+                foss: [false]
+                saml: [false]
+                name: ['']
+                total-concurrency: [5]
```

Comment:
Perhaps a comment here regarding the relationship between total-concurrency and group? I can imagine someone coming in at a later date and just bumping this and therefore missing some groups.

Reply:
Or if there is some way to couple these values together somehow?